src/components/CreatePostInput/index.js

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PropTypes` is added but it's not used in the old hunk. It seems like you are planning to use PropTypes for type checking in your component, which is a good practice for catching bugs by validating the types of props.
```diff
+ import PropTypes from 'prop-types';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->